app/decorators/user_decorator.rb

```diff
@@ -2,8 +2,14 @@ class UserDecorator < ApplicationDecorator
   delegate_all
 
   def cached_followed_tags
-    Rails.cache.fetch("user-#{id}-#{updated_at}/followed_tags", expires_in: 20.hours) do
-      Tag.where(id: Follow.where(follower_id: id, followable_type: "ActsAsTaggableOn::Tag").pluck(:followable_id)).order("hotness_score DESC")
+    Rails.cache.fetch("user-#{id}-#{updated_at}/followed_tags_11-30", expires_in: 20.hours) do
```

Comment:
What's the `_11-30` in the cache key for?

Reply:
It needed a new cache key because the old one should be invalid upon push. 11-30 is today's date. I feel like this is a decent enough way to do it.